engine/plugins/server/server.js

```diff
@@ -25,6 +25,7 @@ const fetch = require('node-fetch');
  */
 module.exports = function(options, imports, register) {
     const bus = imports.bus;
+    const client = imports.client;
```

Comment:
Is it safe to share the client across connections? Have been issue before? May have changed?

Reply:
It shouldn't be a problem to share a plugin... otherwise we have an issue with architect.